engine/src/test/java/io/zeebe/engine/job/FailJobTest.java

```diff
@@ -0,0 +1,262 @@
+/*
```

Comment:
Please move the job tests to `io.zeebe.engine.processor.workflow.job`

Reply:
I think in this test you do not need to activate the jobs before. Jobs can also be failed without activation

Reply:
Could you try to migrate the jobs tests to ClassRules, please?